<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>871</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>871</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PopulationProfiler v1.2</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>5</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>578</width>
         <height>849</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>849</height>
        </size>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>391</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Select analyzed feature</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Select negative control labels</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>248</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>* press [ctrl] or [shift] to 
   select more than one</string>
        </property>
       </widget>
       <widget class="QComboBox" name="groupingFieldCBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>351</y>
          <width>350</width>
          <height>30</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>The well names must have format &quot;CXX&quot;, where: 
&quot;C&quot; is a capital character corresponding to row in the plate,
&quot;XX&quot; is the two digit number of the corresponding column padded with zero if &lt;10.
Examples: &quot;A01&quot;, &quot;F20&quot;, &quot;N14&quot;.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>347</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>The well names must have format &quot;CXX&quot;, where: 
&quot;C&quot; is a capital character corresponding to row in the plate,
&quot;XX&quot; is the two digit number of the corresponding column padded with zero if &lt;10.
Examples: &quot;A01&quot;, &quot;F20&quot;, &quot;N14&quot;.</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="text">
         <string>Select field with well names</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="histogramSettingsGBox">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>442</y>
          <width>521</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>Histogram settings</string>
        </property>
        <widget class="QSpinBox" name="logBaseSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>80</y>
           <width>46</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>22</number>
         </property>
        </widget>
        <widget class="QCheckBox" name="scaleDataCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>251</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Scale data with logarithm of base:</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="useFixedRangeCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>If selected the chosen range will be used to generate the histogram bins. 
Otherwise, the data from the first well will be used to estimate the histogram range. 
The same range is applied to all wells in a given file.</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>Use manual range: </string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="rangeFromSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>50</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Lower limit for the manual histogram range.</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="minimum">
          <double>-999.990000000000009</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="rangeToSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>50</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Upper limit for the manual histogram range.</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="minimum">
          <double>-999.990000000000009</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>25</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Number of bins: </string>
         </property>
        </widget>
        <widget class="QSpinBox" name="noBinsSpinBox">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>20</y>
           <width>46</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>221</x>
           <y>53</y>
           <width>21</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QListWidget" name="negCtrlListWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>220</y>
          <width>350</width>
          <height>121</height>
         </rect>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="analyzedFeatureCBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>391</y>
          <width>351</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Select treatment label field</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>560</y>
          <width>521</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>Histogram analysis</string>
        </property>
        <widget class="QLineEdit" name="manualGatesLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>140</y>
           <width>371</width>
           <height>30</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Histogram bin indexes of the manually selected gates.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="noAnalysisRadioButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>If selected, additional analysis will be performed to 
estimate ditribution of cells in the 5 cell cycle subpopulations: 
&lt;2N, 2N, S, 4N, and &gt;4N.</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>None</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="fixedBinsCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>271</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>PopulationProfiler defines the cell cycle subpopulation boundaries based on the pooled negative control data. 
If this box is selected the same boundaries will be used for all wells. 
Otherwise, the boundaries are adjusted for each well to better fit 2N and 4N peaks.</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>Use fixed bins for cell cycle analysis</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="cellCycleAnalysisRadioButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>401</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>If selected, additional analysis will be performed to 
estimate ditribution of cells in the 5 cell cycle subpopulations: 
&lt;2N, 2N, S, 4N, and &gt;4N.</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>Analyze the cell cycle based on DNA content (automatic gating)</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="selectManualGatesButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>140</y>
           <width>93</width>
           <height>30</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Manual selection of up to 10 gates based on the pooled negative control data.</string>
         </property>
         <property name="text">
          <string>Select gates</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="manualGatingAnalysisRadioButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>401</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>If selected, the histogram will be later divided 
according to manually selected gates.</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>Manual gating</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QComboBox" name="treatmentFieldCBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>180</y>
          <width>350</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="outputPathLineEdit">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>760</y>
          <width>361</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Please, select output path...</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="runButton">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>810</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>play.png</normaloff>play.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="closeButton">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>810</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>exit.png</normaloff>exit.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="selectOutPathButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>760</y>
          <width>160</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Select output folder</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clearInFilesButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>160</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QPushButton" name="selectFilesButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>7</y>
          <width>160</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>select csv files</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>83</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>** all files must have the
   same fields (selected
   below)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>40</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>* each file should contain
   data from a separate plate</string>
        </property>
       </widget>
       <widget class="QListWidget" name="csvFilesListWidget">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>7</y>
          <width>351</width>
          <height>171</height>
         </rect>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>scaleDataCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>logBaseSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useFixedRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>rangeFromSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useFixedRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>rangeToSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cellCycleAnalysisRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>fixedBinsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>638</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualGatingAnalysisRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectManualGatesButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>694</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>724</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualGatingAnalysisRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>manualGatesLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>690</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>720</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noBinsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>manualGatesLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>727</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rangeFromSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>manualGatesLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>736</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rangeToSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>manualGatesLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>744</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logBaseSpinBox</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>manualGatesLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>742</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useFixedRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>manualGatesLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>729</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scaleDataCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>manualGatesLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>729</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
